zh-Hans: Widget translations

JiZPaper opened the pull request.

JiZPaper committed:
Add files via upload

MadeiraAlexandre commented:
Hi, thank you so much! 
I've updated the repo and noticed that Xcode is failing to export the localization of the widgets, but they're already in the file and should be working in the today's build. 
<img width="1552" alt="image" src="https://github.com/user-attachments/assets/ca6e02f0-6f0c-4ca7-baaa-c19c803eb3a9" />
After that I've just added an empty screen that informs the users in case the list is empty. 
<img width="958" alt="image" src="https://github.com/user-attachments/assets/859e3d4c-02d3-45f5-99e2-cf7bf39d1359" />
I'll check how I can make the localization work in the exported files or maybe this is a bug in the current version of Xcode. 

I've also added support for localizing genres as you requested. Thank you so much for your help! 🙏

JiZPaper commented:
> Hi, thank you so much! I've updated the repo and noticed that Xcode is failing to export the localization of the widgets, but they're already in the file and should be working in the today's build. <img alt="image" width="1552" src="https://private-user-images.githubusercontent.com/33880764/430672538-ca6e02f0-6f0c-4ca7-baaa-c19c803eb3a9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDM5MTA5OTAsIm5iZiI6MTc0MzkxMDY5MCwicGF0aCI6Ii8zMzg4MDc2NC80MzA2NzI1MzgtY2E2ZTAyZjAtNmYwYy00Y2E3LWJhYWEtYzE5YzgwM2ViM2E5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDA2VDAzMzgxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU5NTgzMTBhMWU2NWIxNGE3NzI1MGQ1YzZkZGE2MmJhZTBmOGQwN2YyYjI4YmZhYjFmNTVlMGM4NzJmZmM2YzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.lx2KhjDkGhIBPjkjjsFBBcozPzDXKcFtIAihFHn7Vlk"> After that I've just added an empty screen that informs the users in case the list is empty. <img alt="image" width="958" src="https://private-user-images.githubusercontent.com/33880764/430672571-859e3d4c-02d3-45f5-99e2-cf7bf39d1359.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDM5MTA5OTAsIm5iZiI6MTc0MzkxMDY5MCwicGF0aCI6Ii8zMzg4MDc2NC80MzA2NzI1NzEtODU5ZTNkNGMtMDJkMy00NWY1LTk5ZTItY2Y3YmYzOWQxMzU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDA2VDAzMzgxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk1N2QyMTQ1YjM1YWYyM2E4NGQwNzI4NjczMDJmMDc4MDIwY2RiYzlmMDZmOGU5YTA0MTY0YTJlYzU4OTg5NTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8_leyxRf92CN4cFLDjQjZH3FGI6gPQAopADOhdunRaE"> I'll check how I can make the localization work in the exported files or maybe this is a bug in the current version of Xcode.
> 
> I've also added support for localizing genres as you requested. Thank you so much for your help! 🙏

我发现了一个新的问题，在“Rank”中，“Most Popular”是按照添加此动画到列表的用户人数排序的，而“Search”中的“Most Popular”是按照评分高低排序的。它们的排序方式不同而使用了相同的标题，我不知道这样在英语中是否是合理的，但在中文中需要将它们区分开；而且“Rank”中按照评分排序的动画排行榜 标题为“Top All”，而漫画排行榜的标题却是“All”。

Hi, I have noticed a new issue: In the "Rank" section, "Most Popular" is sorted by the number of users who added the anime to their list, while "Most Popular" in the "Search" section is sorted by rating scores. They use the same title despite having different sorting methods. I'm unsure whether this is considered appropriate in English, but in Chinese, these need to be clearly differentiated. Additionally, in the "Rank" section, the anime rankings sorted by ratings are labeled "Top All," whereas the manga rankings use the title "All" for the same category.

MadeiraAlexandre commented:
Hi, thank you for pointing this out. I've fixed the Search ranking that shows in the Search screen for both "Most Popular" anime and manga and also changed to "Top All" title instead of "All" for manga rank.
